### PR TITLE
TRIVIAL: clean up httpclient logging

### PR DESCRIPTION
**Pull request by jdillon**

### CODE REVIEW

**Comment by cstamas** [comment]
+1

File: nexus-core/src/main/java/org/sonatype/nexus/apachehttpclient/Hc4ProviderBase.java [hunk 1]
```diff
@@ -181,7 +171,7 @@ protected void configureAuthentication( final DefaultHttpClient httpClient,
                 authScope = proxyHost.toHostString() + " proxy";
             }
 
-            List<String> authorisationPreference = new ArrayList<String>( 2 );
+            List<String> authorisationPreference = Lists.newArrayListWithExpectedSize(3);
```
Comment (ifedorenko) [inline nexus-core/src/main/java/org/sonatype/nexus/apachehttpclient/Hc4ProviderBase.java:174]: Logging cleanup?

**Comment by ifedorenko** [comment]
+1

File: nexus-core/src/main/java/org/sonatype/nexus/apachehttpclient/Hc4ProviderBase.java (hunk 1 above)
Comment (jdillon) [inline nexus-core/src/main/java/org/sonatype/nexus/apachehttpclient/Hc4ProviderBase.java:174]: Obviously not :-P  I did notice when looking over the logging here that when ntlm was configured that the size is expected to be 3 not 2.  Its a minor and safe optimization which I didn't bother to track in a separate change.

**Comment by adreghiciu** [comment]
+1

**Pull request merged**